docs/resources/flow.md

```diff
@@ -50,5 +50,5 @@ Import is supported using the following syntax:
 
 ```shell
 # Prefect Flows can be imported via flow_id,workspace_id
-terraform import prefect_variable.example 00000000-0000-0000-0000-000000000000,00000000-0000-0000-0000-000000000000
+terraform import prefect_flow.example 00000000-0000-0000-0000-000000000000,00000000-0000-0000-0000-000000000000
```

Comment:
This is just a drive-by fix I saw when working on testing Deployments with Flows.

Reply:
copy pasta strikes again